app/home/home.js

```diff
@@ -1,14 +1,13 @@
-viewsModule.config(function($routeProvider) {
+viewsModule.config(['$routeProvider', function($routeProvider) {
   $routeProvider.when("/", {
     templateUrl : "./home/home.html",
     controller : 'HomeCtrl'
   });
-});
+}]);
 
-viewsModule.controller('HomeCtrl', function($scope, owmUSCities, owmHistory) {
-  owmHistory.push({ name : "Home", isHome : true });
+viewsModule.controller('HomeCtrl', ['$scope', 'owmUSCities', function($scope, owmUSCities) {
```

Comment:
All of the injection arrays are already inserted at minification time by ngmin.

Adding and managing these at development time isn't advised.